pyproject.toml

```diff
@@ -109,11 +109,22 @@ all = [
 dependencies = [
   "pytest~=8.1",
   "pytest-mock~=3.14",
+  "coverage~=7.5.1"
```

Comment:
```suggestion
  "coverage~=7.5.1",
  "pytest~=8.1",
  "pytest-mock~=3.14",
```

Reply:
Syntax highlighting is making this look very ugly, but it's just alphabetising the dependencies :)